### DISCUSSION TITLE
有没有办法在access中获取到当前路由的父路由路径，方便与后端的树型权限数据进行匹配

### DISCUSSION THREAD

**Discussion by xliez** [umijs/umi#12250] [Q&A]
目前 access 返回的方法只接收单个路由的信息作为入参

**Comment by fz6m** [comment]
试试这个 [`useSelectedRoutes`](https://umijs.org/docs/api/api#useselectedroutes) 方法吧，可以拿到当前路径匹配的所有路由信息，倒数第二个位置就是直接的父级，或者用 [`useAppData`](https://umijs.org/docs/api/api#useappdata) ，里面什么都有。
